Dockerfile

```diff
@@ -7,10 +7,16 @@ ENV SONARQUBE_PLUGINS_DIR=/opt/sonarqube/default/extensions/plugins \
     SONARQUBE_WEB_CONTEXT="/sonar" \
     SONARQUBE_FORCE_AUTHENTICATION=true \
     ADOP_LDAP_ENABLED=true \
-    SONARQUBE_JMX_ENABLED=false 
+    SONARQUBE_JMX_ENABLED=false \
```

Comment:
I think this is a type, and needs to be the actual environment variable we're introducing: `SSO_ENABLED`